test/unit/uk/gov/hmrc/bindingtariffclassification/controllers/CaseControllerSpec.scala

```diff
@@ -39,6 +39,7 @@ class CaseControllerSpec extends UnitSpec with WithFakeApplication with MockitoS
 
   private implicit val mat: Materializer = fakeApplication.materializer
 
+  private val c0: Case = CaseData.createCase().copy(reference = null)
```

Comment:
Scala developers do not use `null`.
You should use `Option[String]`

Reply:
I agree with Stefano to not use null (usually), but not really bother if this null is in a test and in control which is the case.